mmv1/templates/terraform/encoders/spanner_instance_partition.go.tmpl

```diff
@@ -0,0 +1,10 @@
+// Temp Logic to accommodate autoscaling_config, processing_units and num_nodes
+if obj["processingUnits"] == nil && obj["nodeCount"] == nil {
+	obj["nodeCount"] = 1
+}
```

Comment:
As `exactly_one_of` is used for  `node_count` and `processing_units`, which means that one of those fields must be provided by the user, it is impossible that both fields are nil.